Modules/Filtering/FFT/src/itkFFTWFFTImageFilterInitFactory.cxx

```diff
@@ -0,0 +1,70 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#include "itkConfigure.h"
+
+#if defined(ITK_USE_FFTWF) || defined(ITK_USE_FFTWD)
+
+#  include "itkFFTWFFTImageFilterInitFactory.h"
+
+#  include "itkFFTWComplexToComplex1DFFTImageFilter.h"
+#  include "itkFFTWComplexToComplexFFTImageFilter.h"
+#  include "itkFFTWForward1DFFTImageFilter.h"
+#  include "itkFFTWForwardFFTImageFilter.h"
+#  include "itkFFTWHalfHermitianToRealInverseFFTImageFilter.h"
+#  include "itkFFTWInverse1DFFTImageFilter.h"
+#  include "itkFFTWInverseFFTImageFilter.h"
+#  include "itkFFTWRealToHalfHermitianForwardFFTImageFilter.h"
+
+#  include "itkCreateObjectFunction.h"
+#  include "itkVersion.h"
+#  include "itkObjectFactoryBase.h"
+
+namespace itk
+{
+FFTWFFTImageFilterInitFactory::FFTWFFTImageFilterInitFactory()
+{
+  FFTWFFTImageFilterInitFactory::RegisterFactories();
+}
+
+FFTWFFTImageFilterInitFactory::~FFTWFFTImageFilterInitFactory() = default;
+
+void
+FFTWFFTImageFilterInitFactory::RegisterFactories()
```

Comment:
This is not called in Python. Perhaps a check could be made that if `Init` is in the name, `RegisterFactories()` is called instead of `RegisterOneFactory()`.